python/in_toto_attestation/predicates/scai/v0/scai.py

```diff
@@ -3,9 +3,8 @@
 import in_toto_attestation.predicates.scai.v0.scai_pb2 as scaipb
 from in_toto_attestation.v1.resource_descriptor import ResourceDescriptor
 
-SCAI_PREDICATE_TYPE = "https://in-toto.io/attestation/scai/attribute-report/"
-SCAI_PREDICATE_VERSION = "v0.2"
-
+SCAI_PREDICATE_TYPE = "https://in-toto.io/attestation/scai/"
+SCAI_PREDICATE_VERSION = "v0.3"
```

Comment:
I wonder if the removed newline here is the issue with the linter?

Reply:
But there still is a newline after the second constant. Let me see if I can get the linter to actually give us a useful error message.

Reply:
I'm not sure if this is configured for PEP8 / a derivative, but top level function and class defs need to be surrounded by two blank lines.

https://peps.python.org/pep-0008/#blank-lines

Reply:
Yeah, you're right. This was the problem. In the future the linter will output the diff for what needs to change.